pkgs/by-name/ni/nixos-rebuild-ng/package.nix

```diff
@@ -3,10 +3,12 @@
   installShellFiles,
   nix,
   nixos-rebuild,
+  nixos-rebuild-ng,
   python3,
+  runCommand,
   withNgSuffix ? true,
 }:
-python3.pkgs.buildPythonApplication {
+python3.pkgs.buildPythonApplication rec {
   pname = "nixos-rebuild-ng";
   version = "0.0.0";
```

Comment:
Well, you can use SemVer to keep track of changes over the Python code.

Reply:
Until we get to an actual first release I am not sure how much sense it would make to arbitrary set versions here.

Even if we do set a version in future right now the code has no way to show those versions (e.g.: we don't have a `--version` flag) unless the user inspect the package, so trying to track a version here is useless for now.